src/components/shared/GoogleAutoComplete/GoogleAutoComplete.tsx

```diff
@@ -11,30 +10,50 @@ import { SETTINGS } from 'configs/settings';
 const { GOOGLE_MAPS_KEY } = SETTINGS;
 
 interface Props {
-  children?: React.ReactNode;
   defaultValue?: string;
   inputRef: React.RefObject<HTMLInputElement>;
-  placesRef: React.RefObject<google.maps.places.SearchBox> | any;
+  onPlaceChange(place: google.maps.places.PlaceResult): void;
 }
 
-interface BoundaryProps {
-  placesRef: React.RefObject<google.maps.places.SearchBox> | any;
-}
+// needed since typescript doesn't recognize 'setFields' as a function
+interface AutocompleteInterface extends google.maps.places.Autocomplete {
+  setFields?(args: String[]): void; 
+} 
+
+class GoogleAutoComplete extends React.Component<Props, any> {
+  autocomplete: AutocompleteInterface;
+  
+  componentDidMount() {
+    if (!this.props.inputRef.current) return;
 
-class StandaloneSearchBoxWithErrorBoundary extends React.Component<BoundaryProps> {
-  state = { hasError: false };
+    this.autocomplete = new google.maps.places.Autocomplete(
+      this.props.inputRef.current,
+      {"types": ["(cities)"]}
```

Comment:
does using the google maps api directly let us prioritize recommended cities? ie, typing in `new` should show `new york` 

I did notice on dev03 typing in `new` showed `newark, new york` vs some random city on beenest.com

Reply:
I think only these types can be used https://developers.google.com/places/supported_types#table3

I'll do a bit more research to see it can be adjusted further. If it can't be changed, another autocomplete library could be an option.

Reply:
we should support cities (new york) and regions (hawaii)